src/components/RewardBudget/RewardBudget.tsx

```diff
@@ -12,11 +12,12 @@ import CheckLine from "src/icons/CheckLine";
 import RewardBudgetBar from "./BudgetBar/RewardBudgetBar";
 import { RewardBudgetUtils } from "./RewardBudget.utils";
 import { Currency } from "src/types";
+import { cn } from "src/utils/cn";
 
 export const RewardBudget: FC<RewardBudgetProps> = props => {
   const { T } = useIntl();
   const [selectedBudget, setSelectedBudget] = useState<WorkEstimationBudgetDetails>(props.budgets[0]);
-  const [amount, setAmount] = useState<number | undefined>(0);
+  const [amount, setAmount] = useState<number | undefined>(undefined);
```

Comment:
I think it would be better practice to handle `number | null`
```suggestion
  const [amount, setAmount] = useState<number | null>(null);
```